draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1846,19 +1848,19 @@ Alternatively, the associated data can be fed into a collision-resistant KDF, su
 
```

Comment:
I think `appending their lengths` => `their little-endian encoded lengths` is useful as an example.

Reply:
Mentioning a very specific encoding feels a bit off, as this is not a specification, just a generic recommendation. A separated text representation of the length, or usage of a serialization format such as Protobuf would be perfectly fine as well. The best representation is application-dependent.
Even in the absence of SHOULD or MUST, mentioning a byte ordering really feels like it's part of a specification, and that it will allow interoperability. But it doesn't.
I think we should just mention the idea of including a length, so that users think about this, and the implications of not having proper separation. And then they can decide the best way to handle the problem for their use case.